vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,7 +1,139 @@
+- Added shortcuts for running internal k9s, crane, jq and fzf (e.g. `ent crane`)
+- `ent attach-kubectx` now supports `--current` to attach the current kubectl context (e.g. following an `oc login`)
+- Modified `get-bundle-id` to support docker-based bundles
+- Fixes to `ent help` and logging
+- `ent check-env` now shows a spinner (can be disabled with `--verbose`) instead of printing the full installation output of npm dependencies 
+- `ent prj` now supports `--from-hub` initialization
+- New variables with overridable defaults: `ENTANDO_CLI_DEFAULT_DOCKER_REGISTRY` and `ENTANDO_CLI_DEFAULT_HUB`
+- Implemented `ent config --effective` to show the effective configuration (composition of hard defaults, distribution defaults and profile configuration)
+- Now jq is always required by `check-env` (both runtime and develop)
+- Experimental support for an enhanced selection menu based on fzf (`ent pkg get fzf`, `ent config --set FZF_SELECT true`)
+- Workarounds to mitigate npm bugs in the update process
+- `ent-diag` now extracts additional details
+
+### Entando Bundle CLI
+
+- New CLI module "ent bundle" to manage docker-based bundles
+- Support for the following workflows:
+   - From scratch: User can create a new bundle project and then add components from existing projects or using external tools
+   - From template: User can create a bundle project by downloading a template from an Entando Hub
+- Complete redesign of the bundle project structure to improve clarity and better separate concerns
+- Clear separation of components (MS, MFE, platform-specific)
+- Clear separation of bundle sources and packaged artifacts
+- A single project file "entando.json" to describe the entire bundle structure, customizations and parameters
+- Support for App Builder EPC extensions
+- Support for all properties of git-based bundle YAML descriptors
+- Support for all properties of docker-based bundles
+- Support for service discovery via the [API claims mechanism](../getting-started/ent-api.md)
+- Source-level support for bundle thumbnails
+- Capability to customize the build, run and packaging commands
+- Custom resource descriptor generation for the current bundle and external Docker repositories
+- Improved log management during parallel builds
+- Improved local testing capabilities and auxiliary services management based on docker-compose
+- Support for custom FTL descriptors (override default automatic generation)
+
+### Entando CMS
+
+- Fixed performance issues caused by unnecessary reprocessing of all Content Types
+
+### Infrastructure / Local Hub
+
+- Entando Component Repository renamed to Local Hub
+- Support for docker-based bundles
+- File structure deployed to AppEngine is redesigned with improved scoping
+- Added new MFE widget capabilities:
+   - Support for service discovery driven by the [API claims mechanism](../getting-started/ent-api.md)
+   - Automatic generation of the widget configUi
```

Comment:
>>configUI

Reply:
@nshaw how is this formatted in code?

Reply:
configUi - https://github.com/entando-samples/e70-react-with-config/blob/main/ui/widgets/widgets-dir/et-widget/bundle/et-widget-descriptor.yaml

Reply:
"Automatic generation of the widget `configUi` integration code"